client/driver/docker.go

```diff
@@ -118,6 +118,27 @@ func createContainer(ctx *ExecContext, task *structs.Task, logger *log.Logger) d
 	logger.Printf("[DEBUG] driver.docker: using %d bytes memory for %s", hostConfig.Memory, task.Config["image"])
 	logger.Printf("[DEBUG] driver.docker: using %d cpu shares for %s", hostConfig.CPUShares, task.Config["image"])
 
+	mode, ok := task.Config["network_mode"]
+        if !ok || mode == "" {
+                // docker default
+                logger.Printf("[WARN] driver.docker: no mode specified for networking, defaulting to bridge")
+                mode = "bridge"
+        }
+
+        // Ignore the container mode for now
+        switch mode {
+        case "default", "bridge", "none":
```

Comment:
Please document the network modes here: https://github.com/hashicorp/nomad/blob/master/website/source/docs/drivers/docker.html.md#client-configuration